ooni/scripts/ooniresources.py

```diff
@@ -38,7 +30,5 @@ def run():
         print "%s: Try --help for usage details." % (sys.argv[0])
         sys.exit(1)
 
-    if options['update-inputs'] or options['update-geoip']:
-        print("WARNING: Passing command line arguments is deprecated")
-
-    yield update.download_resources()
```

Comment:
Uhm, I think that, if the script is *only* deprecated, it should keep downloading resources. Otherwise, you can make it a no-op for backwards compatibility, but in that case the message should be slightly different.